lib/libsnmp/v2.py

```diff
@@ -138,18 +131,19 @@ def snmpTrap(self, remote, trapPDU, community='public'):
         """
         msg = self.createTrapMessage(trapPDU, community)
 
-        self.outbound.put( (msg, remote) )
+        self.outbound.put((msg, remote))
 
     def createSetRequestMessage(self, varBindList, community='public'):
         """ Creates a message object from a pdu and a
             community string.
         """
 
-    def receiveData(self, manager, cb_ctx, (data, src), (exc_type, exc_value, exc_traceback) ):
+    def receiveData(self, manager, cb_ctx, xxx_todo_changeme, xxx_todo_changeme1):
```

Comment:
Since these new `xxx_` variables are used in a few places, it'd be nice to make them more meaningful and help future people understand the code more easily.